docs/copied-from-beats/template-config.asciidoc

```diff
@@ -87,4 +87,16 @@ ifeval::["{beatname_lc}"!="apm-server"]
 
 NOTE: With append_fields only new fields can be added an no existing one overwritten or changed. This is especially useful if data is collected through the http/json metricset where the data structure is not known in advance. Changing the config of append_fields means the template has to be overwritten and only applies to new indices. If there are 2 Beats with different append_fields configs the last one writing the template will win. Any changes will also have an affect on the Kibana Index pattern.
 
+*`setup.template.json.enabled`*:: Set to true to load a json based template file. Specify the path to your Elasticsearch
```

Comment:
this has some implications for the apm server, right? (like breaking the UI)

Reply:
yeah, it's probably not worth trying to explain that either, removed.